pkgs/top-level/python-packages.nix

```
@@ -896,6 +896,8 @@ in {
 
   backports-datetime-fromisoformat = callPackage ../development/python-modules/backports-datetime-fromisoformat { };
 
+  backports_entry-points-selectable = callPackage ../development/python-modules/backports_entry-points-selectable { };
```

```suggestion
  backports-entry-points-selectable = callPackage ../development/python-modules/backports-entry-points-selectable { };
```

pythonPackages should to follow the PEP that converts _ to -.

---

The upstream package is named `backports.entry-points-selectable` which is legal on PyPi, but we can't represent in nix attribute names (except using quoted attribute names) which is why I converted it to an underscore.

`backports-entry-points-selectable` would be a different package on PyPi.

---

> backports-entry-points-selectable would be a different package on PyPi.

really?! I am not sure how to properly to it then without breaking PEP503. https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines

@jonringer how would you do it?

---

Well the contributing guideline clearly states that it should be converted, so I will, I guess. Seems like some other backports* packages need adjusting as well then.

---

We can fix them as soon as we have aliases.

---

Renamed the package, not sure if it is the wisest policy observing PEP503 for upstream packages that don't, but eh.